extensions/Worker.Extensions.Rpc/ci/public-build.yml

```diff
@@ -0,0 +1,51 @@
+# This build is used for public PR and CI builds.
+
+variables:
+  DOTNET_NOLOGO: 1
```

Comment:
As you may need this in many pipelines, feel free to extract a variables template for this repo. I wound up doing that in the host:

https://github.com/Azure/azure-functions-host/blob/dev/eng/ci/templates/variables/build.yml